prisma/seed.ts

```diff
@@ -555,6 +555,16 @@ async function seed() {
       department: "Engineering",
     },
   });
+  await db.profiles.upsert({
+    where: { email: "axel.uzetagomez@in.wizeline.com" },
+    update: {},
+    create: {
+      email: "axel.uzetagomez@in.wizeline.com",
+      firstName: "Axel",
+      preferredName: "Axel",
+      lastName: "Uzeta",
```

Comment:
I think that it's missing the department field

Reply:
It's done 👍